Dockerfile.must-gather

```diff
@@ -1,16 +1,4 @@
-FROM quay.io/openshift/origin-must-gather:4.18.0 AS builder
-
-FROM registry.access.redhat.com/ubi9/ubi-minimal:latest
-RUN microdnf install -y procps-ng tar rsync
-RUN microdnf clean all
-
-# Copy must-gather required binaries
-COPY --from=builder /usr/bin/openshift-must-gather /usr/bin/openshift-must-gather
-COPY --from=builder /usr/bin/oc /usr/bin/oc
-
-# Save original gather script
-COPY --from=builder /usr/bin/gather* /usr/bin/
-RUN mv /usr/bin/gather /usr/bin/gather_original
+FROM quay.io/openshift/origin-cli:4.18.0
```

Comment:
ok, this is now me nitpicking, but still: is this tag pointing to the latest build of 4.18.z or to the latest build of 4.18.0?

Reply:
in addition, can you please link how the `origin-must-gather` image is built (its Dockerfile)?

Reply:
4.X and 4.X.0 have the same content. there are no 4.X.z images in quay:
https://quay.io/repository/openshift/origin-cli?tab=tags&tag=latest

Reply:
I can't find the dockerfile for origin-must-gather but I know that the main must-gather uses the origin-cli base too:
https://github.com/openshift/must-gather/blob/master/Dockerfile.ocp

Reply:
thanks, this looks good enough!

Reply:
so can we use `origin-cli:4.18` like the linked Dockerfile does?

Reply:
dropped this change we don't need it coupled with the base-image update

Reply:
still unaddressed? can we use `4.18` (vs current `4.18.0`)?